CMakeLists.txt

```diff
@@ -12,41 +12,49 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Widgets)
 find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Widgets)
 
-
 include_directories(source/)
+include_directories(source/config)
+include_directories(source/d1formats)
+include_directories(source/dialogs)
+include_directories(source/palette)
+include_directories(source/undostack)
+include_directories(source/views)
+include_directories(source/widgets)
```

Comment:
Maybe it would be nicer to have `#include "config/config.h"` rather then including all the dirs, to me that also makes it more clear how things are structured.

Reply:
Actually, this is a good idea and good catch, I forgot you can specify path in promoted widgets. Let's leave this like this for now, I think it's a good choice for now, later on we can add CMakeLists to the modules.